README.md

```diff
@@ -82,6 +82,7 @@ require'nvim-treesitter.configs'.setup {
   textobjects = {
     move = {
       enable = true,
+      set_jumps = true, -- whether to set jumps in the jumplist
```

Comment:
This should be added in https://github.com/nvim-treesitter/nvim-treesitter-textobjects/blob/111cf356fd5c6c52d2dfb9299a76d201624be3cc/doc/nvim-treesitter-textobjects.txt#L97 as well

Reply:
oh and in the list of supported options https://github.com/nvim-treesitter/nvim-treesitter-textobjects/blob/111cf356fd5c6c52d2dfb9299a76d201624be3cc/doc/nvim-treesitter-textobjects.txt#L85-L94